zwave_js_server/const.py

```diff
@@ -7,6 +7,21 @@
 MAX_SERVER_VERSION = "2.0.0"
 
 
+class LogLevel(IntEnum):
+    """Enum for log levels used by node-zwave-js."""
+
+    # https://github.com/zwave-js/node-zwave-js/blob/master/packages/core/src/log/shared.ts#L12
+    # https://github.com/winstonjs/triple-beam/blame/master/config/npm.js#L14
+    ERROR = 0
+    WARN = 1
+    WARNING = WARN
```

Comment:
According to the above linked NPM levels, `WARNING` is not a thing.

Reply:
I added that in because they are used interchangeably in python, but python doesn't have a `silly` mode so I suppose I should defer to the linked module